source/Octopus.Tentacle.Client/Scripts/Models/ExecuteScriptCommand.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Octopus.Tentacle.Contracts;
+
+namespace Octopus.Tentacle.Client.Scripts.Models
+{
+    public class ExecuteScriptCommand
+    {
+        public ExecuteScriptCommand(
+            ScriptTicket scriptTicket,
+            string taskId,
+            string scriptBody,
+            string[] arguments,
+            ScriptIsolationLevel isolationLevel,
+            TimeSpan isolationMutexTimeout,
+            string isolationMutexName,
```

Comment:
In the process of using this in Server, I found that the mutex name is _always_ set to something, so I changes this to non-nullable.

I also cleaned up these names to be consistent

Reply:
Nit: The three of those:
```
ScriptIsolationLevel isolationLevel,
            TimeSpan isolationMutexTimeout,
            string isolationMutexName,
```

Could be moved to a single class `ScriptIsolation`

Reply:
Added in [8886e99](https://github.com/OctopusDeploy/OctopusTentacle/pull/845/commits/8886e99b2755537ab48270f1a28141562f3ef682)